python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_adapter_config.schema.yml

```diff
@@ -474,13 +474,6 @@ $defs:
               When MPASS is enabled on an MLAG port-channel, MLAG peers coordinate to function as a single PTP logical device.
               Arista PTP enabled devices always place PTP messages on the same physical link within the port-channel.
               Hence, MPASS is needed only on MLAG port-channels connected to non-Arista devices.
-          short_esi:
```

Comment:
Please also clean up some old draft starting in line 434

Reply:
Removed!